src/main/java/org/jetbrains/kotlinx/lincheck/strategy/ManagedStrategyTransformer.java

```diff
@@ -10,31 +10,36 @@
  * it under the terms of the GNU Lesser General Public License as
```

Comment:
It is hard to accept, but it seems that the same code in Kotlin would look much better and safer for further refactorings.